### PR TITLE
Add shadow on hover to all cards.

### PR DESCRIPTION
**Pull request by jayhawk24**
Resolves Issue #306.
Added smooth transition to github cards and also box-shadow in dark mode.
Added box-shadow to achievement cards in dark mode.

### CODE REVIEW

**Review by kartikcho — CHANGES_REQUESTED** [review]
Just need to address one minor thingy and we're good to go!

File: src/components/achievementCard/AchievementCard.css [hunk 1]
```diff
@@ -28,6 +28,9 @@
 .certificate-card:hover {
   box-shadow: rgba(0, 0, 0, 0.2) 0px 20px 30px -10px;
 }
+.dark-mode.certificate-card:hover {
+  box-shadow: rgba(255, 255, 255, 0.2) 0px 20px 30px -10px;
+}
```
[the remark this answers is not in the document]
Comment (jayhawk24) [reply src/components/achievementCard/AchievementCard.css:33]: okay  cool.

**Review by jayhawk24 — COMMENTED** [review]
Works on both modes (light and dark)

**Review by kartikcho — APPROVED** [review]
Everything looks good, a great PR! Thanks!

**Comment by kartikcho** [comment]
@allcontributors please add @jayhawk24 for code

**Pull request merged**